apidoc/Titanium/Platform/Platform.yml

```diff
@@ -407,3 +407,34 @@ properties:
   - name: irq
     summary: The number of milliseconds the CPU has spent in irq mode.
     type: Number
+
+---
+name: OpenURLOptions
+summary: |
+    The optional options to pass to the URL handling (iOS 10+). Pass a
+    dictionary with one or more of the following string-keys:
+        * `UIApplicationOpenURLOptionsSourceApplicationKey` (String value)
+        * `UIApplicationOpenURLOptionsAnnotationKey` (Array value)
+        * `UIApplicationOpenURLOptionsOpenInPlaceKey` (Boolean value)
+        * `UIApplicationOpenURLOptionUniversalLinksOnly` (Boolean value)
+    Read more about the available keys in the [Apple documentation](https://developer.apple.com/documentation/uikit/uiapplicationopenurloptionskey?language=objc).
+properties:
+  - name: UIApplicationOpenURLOptionsSourceApplicationKey
+    type: String
+    summary: The value of this key is an String containing the bundle ID of the app that made the request. If the request originated from another app belonging to your team, UIKit sets the value of this key to the ID of that app. If the team identifier of the originating app is different than the team identifier of the current app, the value of the key is nil.
+    optional: true
+
+  - name: UIApplicationOpenURLOptionsAnnotationKey
+    type: Array
```

Comment:
Here must be a type of items stored in the array.

Reply:
Yeah... I couldn't find what the heck goes in here.

Reply:
FYI: with `type: Array` current version of the "typescript" generator will produce incorrect output.